apps/lwipserver2/include/echo/tuning_params.h

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2020, Data61, CSIRO (ABN 41 687 119 230)
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+/* Number of buffers used for sending and receiving ethernet frames. */
+#define TX_BUFS 510
+#define RX_BUFS 510
+
+/* Size used for ethernet buffers */
```

Comment:
PLease add the note again that this is the next 2^n for the 1500 byte MTU. I assume that this buffer size is independent of the buffer size that ius set it the other PRs, might be good to mention this in the comment also, so it's clear there are not pitfalls in here.